### PR TITLE
Use ABI-specific integers from `dart:ffi`

### PR DESCRIPTION
**Pull request by simolus3**
Move imports from `package:ffi` back to `dart:ffi`, bump version to 5.0.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 5.0.0
+
+- Stable release targeting Dart 2.17, supporting ABI-specific integer types.s
```
Comment (dcharkes) [inline CHANGELOG.md:3]: typo at the end of the sentence.

Comment (mannprerak2) [inline CHANGELOG.md:3]: leading `s`

**Review by dcharkes — COMMENTED** [review]
Can we remove the dependency on `package:ffi` completely?

**Review by mannprerak2 — APPROVED** [review]
LGTM

**Comment by simolus3** [comment]
> Can we remove the dependency on `package:ffi` completely?

The ObjC bindings seem to use `calloc` from that package. I'll try to make this package not emit unused imports.

**Comment by dcharkes** [comment]
Sounds good to me.

`calloc` is available in 1.1.2 as well, so we can depend on that instead.

File: lib/src/header_parser/parser.dart [hunk 2]
```diff
@@ -70,8 +70,9 @@ List<Binding> parseToBindings() {
   }
 
   _logger.fine('CompilerOpts used: $compilerOpts');
-  clangCmdArgs = createDynamicStringArray(compilerOpts);
-  final cmdLen = compilerOpts.length;
+  clangCmdArgs = createDynamicStringArray(
+      [...compilerOpts, '-I', '/usr/lib/clang/13.0.1/include/']);
```
Comment (dcharkes) [inline lib/src/header_parser/parser.dart]: Can this path be found automatically?

For example with https://github.com/dart-lang/ffigen/blob/master/lib/src/config_provider/path_finder.dart

Also, I believe the headers that we use are in https://github.com/dart-lang/ffigen/tree/master/third_party/libclang/include/clang-c

**Review by dcharkes — APPROVED** [review]
Thanks for the PR Simon!

File: lib/src/header_parser/parser.dart (hunk 2 above)
Comment (dcharkes) [reply lib/src/header_parser/parser.dart]: Oh, you already beat me to it! 😄

**Comment by dcharkes** [comment]
Thanks @simolus3 ! 🚀

**Pull request merged**